Fix guardicore redirection links (#2912)
guardicore/monkey: README.md

  * WMI
  * Log4Shell
  * Zerologon

>   * and more, see our [Documentation hub](https://techdocs.akamai.com/infection-monkey/docs/exploiters/) for more information about our RCE exploiters.

Reviewer: ```suggestion
  * and more, see our [documentation hub](https://techdocs.akamai.com/infection-monkey/docs/exploiters/) for more information.
```

## Setup

> Check out the [Setup](https://techdocs.akamai.com/infection-monkey/docs/setting-up-infection-monkey/) page in the Wiki or a quick getting [started guide](https://techdocs.akamai.com/infection-monkey/docs/getting-started/).

Reviewer: ```suggestion
Check out the [Setup](https://techdocs.akamai.com/infection-monkey/docs/setting-up-infection-monkey/) page and the [Getting Started](https://techdocs.akamai.com/infection-monkey/docs/getting-started/) guide in our documentation.
```

Reviewer: We should update the links in the Wiki too. Or just remove the Wiki completely.
